### DISCUSSION TITLE
如何隐藏默认空数据

### DISCUSSION THREAD

**Discussion by Create-Peace** [antvis/S2#2595] [Q&A]
如果传入数据为空，会默认渲染一行空数据，如何去掉这个默认空数据？
![image](https://github.com/antvis/S2/assets/12973954/4fd9b625-f3fa-4773-88ef-34d616a3e06c)

**Comment by lijinke666** [comment]
透视表默认会有一个占位骨架, 可以自行动态处理下 https://s2.antv.antgroup.com/manual/advanced/custom/cell-size#%E8%B0%83%E6%95%B4%E8%A1%8C%E5%A4%B4%E5%8D%95%E5%85%83%E6%A0%BC%E5%AE%BD%E9%AB%98

```ts
const s2Options = {
  style: {
    rowCell: {
      height(node) {
        return !node?.value ? 0 : 100;
      },
    }
  }
}
```

![image](https://github.com/antvis/S2/assets/21015895/f679537d-4fda-499d-bf72-d24c09a179b0)
